packages/experimental/Button/MIGRATION.md

```diff
@@ -0,0 +1,52 @@
+# Button Migration
+
+## STATUS: WIP 🚧
+
+This Migration guide is a work in progress and is not yet ready for use.
+
+## Migration from v0 Button
+
+### Component renames
+
+Common buttons now all map to `Button`:
+
+| v0 `Button`                             | v1 `Button`                                          |
+| --------------------------------------- | ---------------------------------------------------- |
+| `<Button text="Hello, world" />`        | `<Button>Hello, world</Button>`                      |
+| `<PrimaryButton text="Hello, world" />` | `<Button appearance='primary'>Hello, world</Button>` |
+| `<StealthButton text="Hello, world" />` | `<Button appearance='subtle'>Hello, world</Button>`  |
+
+### Props that remain as is
+
+- `children`
+- `disabled`
+- `icon`
+- `onClick`
+- `testID`
+- `tooltip`
+
+### Props no longer supported with an equivalent functionality in v1 Button
+
+- `componentRef` => Use regular `ref` instead
+- `content` => Pass the content as `children` instead
+- `endIcon` => Use `right` value for `iconPosition` prop and pass icon information into `icon` prop instead
+- `startIcon` => Use `left` value for `iconPosition` prop and pass icon information into `icon` prop instead
+
+### Props no longer supported without an equivalent functionality in v1 Button
+
+- Cannot use both `startIcon` and `endIcon` at the same time in v1
```

Comment:
What will MenuButton do, which has both icons?

Reply:
The chevron will be a special case. The main argument against the additional flexibility that was introduced with having icons on both ends is that design isn't speccing for that case, so we can't guarantee that it will look nice/follow fluentui designs. (and also alignment with web). The chevron is specced to occupy a smaller amount of space so it's a special case.